example/lib/pages/overlay_image.dart

```diff
@@ -15,7 +14,7 @@ class OverlayImagePage extends StatelessWidget {
     final bottomRightCorner = LatLng(53.475, 0.275);
     final bottomLeftCorner = LatLng(52.503, -1.868);
 
-    final overlayImages = [
+    final overlayImages = <OverlayImage>[
```

Comment:
This needs to be changed to support #1315.

```suggestion
    final overlayImages = <BaseOverlayImage>[
```